aion/web/elm/src/General/Constants.elm

```diff
@@ -24,3 +24,14 @@ panelPath =
 userPath : String
 userPath =
     "#profile"
+
+
+footerContent : List String
```

Comment:
See, if you put that in the `View` file it would really suck a lot. Here it fits much better. 

The footer could definitely be shorter, but I would take care of that in a separate PR that would deal with other imperfect strings we display (e.g. saying "Error" when someone sends wrong answer).